ambari-common/src/main/python/resource_management/libraries/script/script.py

```diff
@@ -781,6 +781,15 @@ def load_available_packages(self):
 
     service_name = config['serviceName'] if 'serviceName' in config else None
     repos = CommandRepository(config['repositoryFile'])
+
+    from resource_management.libraries.functions import lzo_utils
+
+    # remove repos with 'GPL' tag when GPL license is not approved
+    repo_tags_to_skip = set()
+    if not lzo_utils.is_gpl_license_accepted():
+      repo_tags_to_skip.add("GPL")
+    repos.items = [r for r in repos.items if not (repo_tags_to_skip & r.tags)]
```

Comment:
Is bitwise and operator deliberate here?

Reply:
See my comment below